<launch>

  <!--                             -->
  <!-- Jogging nodes for both arms -->
  <!--                             -->

	<rosparam command="load" file="$(find teach_motions)/cfg/sim_right_jog_settings.yaml" ns="right_arm" />
	<node name="right_jog_arm_server" pkg="jog_arm" type="jog_arm_server" output="screen" >
	    <param name="parameter_ns" type="string" value="right_arm" /> 
	</node>

	<rosparam command="load" file="$(find teach_motions)/cfg/sim_left_jog_settings.yaml" ns="left_arm" />
	<node name="left_jog_arm_server" pkg="jog_arm" type="jog_arm_server" output="screen" >
	    <param name="parameter_ns" type="string" value="left_arm" /> 
	</node>


  <!--                                      -->
  <!-- For quick testing of the arm jogging -->
  <!--                                      -->
<!--
  <node name="spacenav" pkg="spacenav_node" type="spacenav_node" />
  <node name="spacenav_to_twist" pkg="teach_motions" type="spacenav_to_twist.py" >
    <remap from="jog_arm_server/delta_jog_cmds" to="jog_arm_server/left_arm/delta_jog_cmds" />
  </node>
-->

  <!--                                              -->
  <!-- Read the datafiles and send jogging messages -->
  <!--                                              -->

  <rosparam command="load" file="$(find teach_motions)/cfg/dual_arm_husky.yaml" />

  <!-- Typically run this separately after the jogging nodes have time to initialize -->
<!--  <node name="read_and_send" pkg="teach_motions" type="read_and_send" output="screen" /> -->

</launch>
